### DISCUSSION TITLE
New line / Center in text or variable

### DISCUSSION THREAD

**Discussion by knallt** [sidoh/epaper_templates#66] [Q&A]
Hey, 

is it possible to user wordwraps in static text or variables? I tried "\n" ... this doesn't work. 
Centering of the text is also not possible or?

Thanks you so much!

**Comment by sidoh** [comment]
Currently no special handling for newline characters, but would hopefully be very easy to add (1-line change) here:

https://github.com/sidoh/epaper_templates/blob/master/lib/Display/TextRegion.cpp

Happy to review a PR if you're feeling up to it. :)

Text alignment is trickier because Adafruit GFX does not support it natively. Would basically need to implement it from scratch.

**Comment by knallt** [comment]
I guess this is beyond my abilities ;)
Thank you so much!
